cdap-ui/app/cdap/components/Management/index.js

```diff
@@ -33,7 +33,7 @@ var shortid = require('shortid');
 var classNames = require('classnames');
 
 var dummyData = {
-  version: '4.0-SNAPSHOT',
+  version: '4.0.0',
```

Comment:
this need not be dummy, right? We should use http://docs.cdap.io/cdap/current/en/reference-manual/http-restful-api/configuration.html#cdap-version